templates/support/index.html

```diff
@@ -139,578 +171,747 @@ <h2 class="p-heading--4">Security, support and managed open source for enterpris
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item u-sv1">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/56d8741e-node-logo.png",
+              alt="",
+              width="196",
+              height="253",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item u-sv1">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/fa78f5b4-openjdk logo.png",
+            alt="",
+            width="240",
+            height="253",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+          </div>
+          <div class="p-logo-section__item u-sv1">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/05c01589-redis logo.png",
+              alt="",
+              width="252",
+              height="253",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item u-sv1">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/850af73c-ruby-rails logo.png",
+              alt="",
+              width="240",
+              height="253",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+        </div>
+      </div>
     </div>
-    <div class="col-6 p-divider__block">
-      <h4>Infrastructure</h4>
-      <p>Security and IAAS support for open source infrastructure.</p>
-      <br />
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Security patching for more than 2,500 packages in Ubuntu Main repository (after 5 years of free LTS security patching)</li>
-        <li class="p-list__item is-ticked">10 years support for Ubuntu LTS releases</li>
-        <li class="p-list__item is-ticked">Bug-fix and operational support for <a href="/openstack">OpenStack</a>, KVM, MAAS, LXD, Ceph</li>
-      </ul>
+    <div class="col">
+      <h5>Add phone and ticket support on top of Ubuntu Pro</h5>
+      <p>Build with confidence with 24/7/365 phone and ticket support. Get prompt help when something breaks on more than 25,000 packages in the Ubuntu Main and Universe repositories, including the most widely used open source applications and toolchains.</p>
+      <p>Benefit from the collective knowledge of our team, based on our 20 years of experience building and sustaining open source software.</p>
     </div>
   </div>
-  <div class="row u-equal-height">
-    <div class="col-6">
-      <p>
-        <a href="/support#apps-support">Supported applications&nbsp;&rsaquo;</a>
-      </p>
+</section>
+
```

Comment:
please leave only one line between sections and make it consistent throughout the whole file